lib/next_ls/runtime/sidecar.ex

```diff
@@ -31,4 +54,60 @@ defmodule NextLS.Runtime.Sidecar do
 
     {:noreply, state}
   end
+
+  defp filter_attributes(symbols) do
+    symbols
+    |> Enum.filter(&match?({:attribute, _, _, _}, &1))
+    |> Enum.reject(fn {_, "@" <> name, _, _} ->
+      Map.has_key?(Module.reserved_attributes(), String.to_atom(name))
+    end)
```

Comment:
For optimum efficiency, i'd combine these so its operated in a single pass

Reply:
yes, agree. Will fix that.